pkg/apis/componentconfig/types.go

```diff
@@ -20,6 +20,191 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ClientConnectionConfiguration contains details for constructing a client.
```

Comment:
These should be removed from this file as #53645 is merged.

Reply:
Fixed. Thanks!